Xlib/protocol/display.py

```diff
@@ -25,6 +25,9 @@
 import errno
 import socket
 
+# Python 2/3 compatibility.
+from six import PY3, byte2int, indexbytes
```

Comment:
Sorry, maybe this is not a PEP8 recommendation but Landscape.io or QuantifiedCode warning (I can't find it quickly right now).